src/rapid_pro_to_engagement_db/rapid_pro_to_engagement_db.py

```diff
@@ -194,6 +198,8 @@ def sync_rapid_pro_to_engagement_db(rapid_pro, engagement_db, uuid_table, flow_r
             contact_urn = contact.urns[0]
             participant_uuid = _de_identify_contact_urn(contact_urn, uuid_table)
 
+            test_run = run.contact.uuid in test_contacts
```

Comment:
I'm not sure about this strategy because it means we have to have the test contact uuids correct when we first copy the message to the database, and if we change who we consider to be a test contact (e.g. because someone at avf tested the line without checking with us first) that could be quite expensive.

Perhaps it would make more sense to change the test contacts uuid configuration to refer to participant uuids rather than rapid pro contact uuids. Then it's super easy to change who we consider test contacts, and we can more easily copy these across projects where these ids don't change (unlike rapid pro, which assigns a new uuid in every workspace).

Reply:
Thanks for noting this. I see how filtering using participant uids and in analysis is a better approach than in rapidpro fetch as it used to be. Have removed the filters, for now, will filter test participants after merging the messages traced data in individuals traced data.